core/src/tests/legacy.spec.ts

```diff
@@ -102,8 +102,8 @@ describe('legacy', () => {
 
   it('doc.addEventListener backbutton', done => {
     const AppWeb = class {
-      async addListener(eventName: string) {
-        expect(eventName).toBe('backButton');
+      async addListener(event: any) {
+        expect(event.eventName).toBe('backButton');
```

Comment:
Why was this change necessary? `addListener()` methods within plugin classes shouldn't have this signature.

Reply:
when I added the androidBridge to the window object (line119), it changed the result that was coming back from the method from just a string to the object.